src/test/java/com/yahoo/bullet/storm/FilterBoltTest.java

```diff
@@ -613,7 +667,6 @@ public void testGroupAllCount() {
 
         Tuple nonMatching = makeRecordTuple(another);
         IntStream.range(0, 5).forEach(i -> bolt.execute(nonMatching));
-        bolt.execute(nonMatching);
```

Comment:
?

Reply:
I don't understand the point of the original code. It executes 5 nonMatching in the line above. Why add a separate one beneath..?

Reply:
There was a reason but I can't remember why now :derp: